src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -44772,10 +44772,10 @@ static void F(string str)
             Assert.Equal(NullableAnnotation.Annotated, symbol.TypeWithAnnotations.NullableAnnotation);
 
             var declaration = tree.GetRoot().DescendantNodes().OfType<VariableDeclarationSyntax>().First();
-            Assert.Equal("System.String?", model.GetTypeInfo(declaration.Type).Type.ToTestDisplayString());
-            Assert.Equal(CodeAnalysis.NullableAnnotation.Annotated, model.GetTypeInfo(declaration.Type).Nullability.Annotation);
-            Assert.Equal("System.String?", model.GetTypeInfo(declaration.Type).ConvertedType.ToTestDisplayString());
-            Assert.Equal(CodeAnalysis.NullableAnnotation.Annotated, model.GetTypeInfo(declaration.Type).ConvertedNullability.Annotation);
+            Assert.Equal("System.String?", model.GetTypeInfoAndVerifyIOperation(declaration.Type).Type.ToTestDisplayString());
+            Assert.Equal(CodeAnalysis.NullableAnnotation.Annotated, model.GetTypeInfoAndVerifyIOperation(declaration.Type).Nullability.Annotation);
```

Comment:
>GetTypeInfoAndVerifyIOperation [](start = 74, length = 30)

Perhaps the second time we can skip IOperation check? There are several more similar cases like this.

Reply:
Copied comment back in line.

---
In reply to: [402671474](https://github.com/dotnet/roslyn/pull/42403#discussion_r402671474) [](ancestors = 402671474)